lib/components/settings/layout/layout_settings_screen.dart

```diff
@@ -28,11 +30,14 @@ class _LayoutSettingsScreenState extends State<LayoutSettingsScreen> {
           setState(() => layoutPreferences!.compactMode = value);
         },
       ),
-      const ListTile(
-        leading: Icon(CupertinoIcons.textformat),
-        title: Text('Font'),
-        subtitle: Text('coming soon!'),
-        enabled: false,
+      ListTile(
+        leading: const Icon(CupertinoIcons.textformat),
+        title: const Text('Font'),
+        onTap: () {
+          app<HarpyNavigator>().pushNamed(
+            FontSettingsScreen.route,
+          );
+        },
```

Comment:
I think the font settings should be in the main settings screen now instead of being in the layout setting screen